### PR TITLE
Added assert functions and checking additional attributes.

### PR DESCRIPTION
**Pull request by cmumford**
Added assertEquals, assertTrue, and assertFalse to simplify
value checks in the test.

Also, added missing checks for object attributes.

### CODE REVIEW

File: characteristic_notify/web_app.js [hunk 1]
```diff
@@ -108,12 +105,17 @@ async function startTest() {
...
 
     logInfo(`Connected to GATT, requesting service: ${testService}...`);
     const service = await gattServer.getPrimaryService(testService);
+    assertEquals(service.device, device, 'service device mismatch');
 
-    logInfo(`Got service, requesting characteristic ${testCharacteristic}...`);
+    logInfo(`Connected to service uuid:${service.uuid}, primary:${service.isPrimary}`);
+    logInfo(`Requesting characteristic ${testCharacteristic}...`);
     const characteristic = await service.getCharacteristic(testCharacteristic);
+    assertEquals(service.device, device, 'Characteristic service mismatch');
```
Comment (reillyeon) [inline characteristic_notify/web_app.js]:
```
assertEquals(characteristic.service, service, 'characteristic service mismatch');
```

**Review by reillyeon — APPROVED** [review]
LGTM with a copy/paste error fixed.

File: characteristic_notify/web_app.js (hunk 1 above)
Comment (cmumford) [reply characteristic_notify/web_app.js]: fixed.

Comment (reillyeon) [reply characteristic_notify/web_app.js]: Change service to device as well.

**Pull request merged**